src/components/Notification/Notification.js

```diff
@@ -21,7 +21,7 @@ const Notification = ({
   const timeoutId = useRef();
 
   useEffect(() => {
-    if (timeout && close && !timeoutId.current) {
+    if (timeout && close) {
```

Comment:
So I *think* the issue here was that the timeout was being created then immediately cleared (useEffect weirdness?), but since `timeoutId.current` was no longer falsy it didn't create a new one.